core/src/test/scala/precog/AutoBumpSpec.scala

```diff
@@ -129,7 +129,33 @@ class AutoBumpSpec(params: CommandLine) extends Specification with ScalaCheck wi
   }
 
   "draftPullRequest" should {
-    val updateBranch = UpdateBranch(Runner.DefaultConfig, "trickle/branch", None, List("Something old, something new"), Some(Revision))
+    val updateBranch =
```

Comment:
Most test cases added are not actually needed because this particular value isn't used, but I decided it was best to treat the code as a black box.